app/config/message-queue.js

```diff
@@ -0,0 +1,52 @@
+const joi = require('joi')
+const msgTypePrefix = 'uk.gov.ffc.ahwr'
+
+const sharedConfigSchema = {
+  appInsights: joi.object(),
+  host: joi.string(),
+  password: joi.string(),
+  username: joi.string(),
+  useCredentialChain: joi.bool().default(false)
+}
+
+const schema = joi.object({
+  sfdMessageRequestQueue: {
+    address: joi.string(),
+    type: joi.string(),
+    ...sharedConfigSchema
+  },
+  sfdMessageTopic: {
+    address: joi.string(),
+    ...sharedConfigSchema
+  },
+  sfdMessageRequestMsgType: joi.string()
+})
+
+const sharedConfig = {
+  appInsights: require('applicationinsights'),
+  host: process.env.MESSAGE_QUEUE_HOST,
+  password: process.env.MESSAGE_QUEUE_PASSWORD,
+  username: process.env.MESSAGE_QUEUE_USER,
+  useCredentialChain: process.env.NODE_ENV === 'production'
+}
+
+const config = {
+  sfdMessageRequestQueue: {
+    address: process.env.SFDMESSAGEREQUEST_QUEUE_ADDRESS,
+    type: 'queue',
+    ...sharedConfig
+  },
+  sfdMessageTopic: {
```

Comment:
Is this the real topic, or the proxy/mock one? Maybe worth allowing both to be specified, and then having a toggle between the two.

Reply:
Resolving for now, hold over for future pr